bin/run-tests-on-swarm.sh

```diff
@@ -179,7 +179,13 @@ readonly BASEDIR=/tmp/testresults
 readonly NETWORK="$USER-vespa"
 readonly TESTRUNNER="$USER-testrunner"
 readonly SERVICE="$USER-vespanode"
-readonly VESPAVERSION=$(docker run --rm $BINDMOUNT_OPTS --entrypoint env $DOCKERIMAGE /opt/vespa/bin/vespa-print-default version)
+VESPAVERSION=$(docker run --rm $BINDMOUNT_OPTS --entrypoint bash $DOCKERIMAGE -lc '${VESPA_HOME-/opt/vespa}/bin/vespa-print-default version')
```

Comment:
Need to use double quotes to get $VESPA_HOME evaluated in this scripts context.

Reply:
The VESPA_HOME environment variable is supposed to be picked up from the image and be expanded by the bash process running in the container.

Reply:
Right.